include/swift/Basic/LangOptions.h

```diff
@@ -331,6 +331,9 @@ namespace swift {
     /// `@differentiable` declaration attribute, etc.
     bool EnableExperimentalDifferentiableProgramming = false;
 
+    /// Whether to enable forward mode differentiation
```

Comment:
Please surround newly added code in `// SWIFT_ENABLE_TENSORFLOW` and `// SWIFT_ENABLE_TENSORFLOW END`.

Please do this only for `tensorflow` branch specific changes, like `EnableExperimentalForwardModeDifferentiation` - not `EnableExperimentalDifferentiableProgramming`, which has already been upstreamed to `master`.

Reply:
Context: `// SWIFT_ENABLE_TENSORFLOW` (and `END`) help us identify which code snippets are `tensorflow`-specific - this is useful for upstreaming code and fixing merge conflicts from `master`.

`// SWIFT_ENABLE_TENSORFLOW` is necessary only in files that exist upstream on `master`, not in `tensorflow` branch newly added files.